lib/src/revset.rs

```diff
@@ -1659,6 +1659,24 @@ fn resolve_branch(repo: &dyn Repo, symbol: &str) -> Option<Vec<CommitId>> {
     None
 }
 
+fn collect_branch_symbols(repo: &dyn Repo, include_synced_remotes: bool) -> Vec<String> {
```

Comment:
(No action needed) It might be nice to extract the common functionality between this and `export_some_refs`.

Reply:
I'm not sure how we can do, but I tend to agree that we have similar pattern in various places.